lib/drivers/medtronic600/NGPUtil.js

```diff
@@ -52,6 +52,16 @@ class NGPTimestamp {
 }
 
 class NGPConstants {
+  static get PUMP_STATE_FLAGS() {
+    return {
+      SUSPENDED: 1,
+      BOLUSING: 2,
+      ACTIVE: 16, // INACTIVE means pump rewound, needs reservoir
```

Comment:
It looks like each of these flags is one bit that's either on or off. So if `ACTIVE` is 0 it's `INACTIVE`? Any idea what is stored in the two bits between `BOLUSING` and `ACTIVE`?

Reply:
Yes, it's a bitfield, so bit 4 being 0 would be INACTIVE. That's a term I came up with myself, because I had to reverse engineer these status bits. I haven't figure out what the missing bits are yet...